Makefile

```diff
@@ -63,6 +63,20 @@ CRATES := \
 	$(wildcard */*/*/*/Cargo.toml) \
 	$(wildcard */*/*/*/*/Cargo.toml)
 
+ALLMAKEFILE := \
+	$(wildcard */Makefile) \
+	$(wildcard */*/Makefile) \
+	$(wildcard */*/*/Makefile) \
+	$(wildcard */*/*/*/Makefile) \
+	$(wildcard */*/*/*/*/Makefile)
```

Comment:
`$(shell find -name Makefile -type f)

Reply:
we actually need to make that top level [a-z]* it turns out. The find will find too many things. but it will ignore 3rdparty